contracts/standard/arbitration/ArbitrableDeposit.sol

```diff
@@ -0,0 +1,80 @@
+
+/**
+ *  @title ArbitrableDeposit
+ *  @author Luke Hartman - <lhartman3@zagmail.gonzaga.edu>
+ *  Bug Bounties: This code hasn't undertaken a bug bounty program yet.
+ */
+
+
+pragma solidity ^0.4.15;
+import "./TwoPartyArbitrable.sol";
+
+
+/** @title Arbitrable Deposit
+ *  This is a a contract for a deposit by the owner where a claimant can seek arbitration. 
+ *  Party A is the owner and Party B is the claimant.
+ */
+contract ArbitrableDeposit is TwoPartyArbitrable {
+    string constant RULING_OPTIONS = "Allow partyA deposit;Pay partyB";
+    uint public amount; // Amount deposited by owner
+
+    uint public claimAmount; // Amount claimed by claimant
+    uint public claimRate; // Rate of a claim the claimant must deposit as an integer
+    uint internal claimResponseAmount; // Amount the claimant is granted by the owner
+
+    function ArbitrableDeposit (
+        Arbitrator _arbitrator, bytes32 _hashContract, uint _timeout,
+        address _partyB, bytes _arbitratorExtraData)
```

Comment:
In this case partyB is specified, so it's not 

> Any party may make a claim

We must allow anyone to make claims, so this contract will probably not inherit from TwoPartyArbitrable.